docs/telemetry.md

```diff
@@ -22,32 +121,148 @@ An example of the data collected is the following:
 {"key" : "hardware_arch","value" : "x86_64 x86_64"}]}]
 ```
 
+## Telemetry Agent
+
+The Percona Telemetry agent functions as a dedicated OS daemon process. The service name on the operating system is `percona-telemetry-agent`. The agent can create, read, write, and delete files in the `${telemetry root path}` and only interacts with JSON files.
+
+The agent sends the collected information as a file to the Percona Platform every 24 hours. If the attempt fails, the agent tries again after a defined period with a limit of five attempts. After a successful transmission, the agent deletes the file.
+
+The agent does not send anything if no Percona-specific files are in the target directory.
```

Comment:
Is that true? No OS-level metrics sent?

Reply:
That's what is described in the project document.